app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(cafe: Cafe, friends: list[dict]) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (
+            VaccineError,
+            NotWearingMaskError
+        ) as e:
+            if isinstance(e, VaccineError):
+                return "All friends should be vaccinated"
+            elif isinstance(e, NotWearingMaskError):
+                masks_to_buy += 1
```

Comment:
Write two except blocks and do not process errors using if conditions, you can manage it in other way
```suggestion
        except VaccineError:
            ....
        except NotWearingMaskError:
            ....
```